id_infrastructure/firestore_uuid_table.py

```diff
@@ -21,6 +21,43 @@ def __init__(self, table_name, crypto_token_path, uuid_prefix):
         self._table_name = table_name
         self._uuid_prefix = uuid_prefix
 
+    def set_mappings(self, mappings):
```

Comment:
Consider 'update_data_to_uuid_mappings'

This is not a function that we should be using on a regular basis so I don't mind the ugly name and would rather be very explicit

Reply:
changed